### PR TITLE
Toggle

### PR DESCRIPTION
**Pull request by annadragan**

### CODE REVIEW

**Review by Zaid0307 — COMMENTED** [review]
Good Job, its a lot of Code 😄

File: src/pages/VocabularyCards.js [hunk 1]
```diff
@@ -17,15 +18,30 @@ export default function VocabularyCards({
...
       {vocabulary.length === 0 ? (
         <>
           <Text>
-            Das ist ein besonderer Ort, da du diesen nach eigenen Vorliben
-            gestallten kannst. Hier kannst du die für dich wichtigen Vokabeln
-            und Sätze aufbewahren und trainieren. Die Vokabelkärtchen mit den
-            für dich persönlich interessanten und spanennden Wörtern sind beim
-            perfektionieren einer anderen Sprache der Schlüssel zum Erfolg!
+            Das ist ein besonderer Ort, da du diesen nach deinen eigenen
+            Vorlieben gestallten kannst. Hier kannst du die für dich wichtigen
```
Comment (Ju-Schn) [inline src/pages/VocabularyCards.js:40]:
```suggestion
            Vorlieben gestalten kannst. Hier kannst du die für dich wichtigen
```

**Review by Ju-Schn — APPROVED** [review]
Nice work ;)

Just a few comments :)

File: src/components/Form.js [hunk 2]
```diff
@@ -25,9 +26,23 @@ export default function Form({ onCreateTerm }) {
...
+        <Break
+          style={{
+            wordBreak: 'break-word',
+          }}
+        ></Break>
+        Formular
+      </Header>
       <FormWrapper
-        aria-describedby="Create a new word"
+        aria-describedby="Neues Wort erstellen"
```
[the remark this answers is not in the document]
Comment (macfrei) [reply src/components/Form.js:45]: `aria-describedby` actually should get an `id` and not a text, I think 🤔 Have a look here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby

**Pull request merged**